docs/networking.md

```diff
@@ -121,6 +121,41 @@ $ kops create cluster \
 
 The above will deploy a daemonset installation which requires K8s 1.4.x or above.
 
+##### Enable Cross-Subnet mode in Calico
+Calico [since 2.1] supports a new option for IP-in-IP mode where traffic is only encapsulated
```

Comment:
yes! :) BGP is used to broadcast node-specific routes to all the nodes on the cluster.